src/main/java/ita/softserve/course_evaluation/dto/UserDtoMapper.java

```diff
@@ -0,0 +1,52 @@
+package ita.softserve.course_evaluation.dto;
+
+import ita.softserve.course_evaluation.entity.User;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+
+@Component
```

Comment:
Since all the methods are static - we do not need this as a Bean. Remove ```@Component```

Reply:
removed